exampleCourse/questions/element/richTextEditor/question.html

```diff
@@ -15,7 +15,7 @@
 
 <pl-hide-in-panel answer="true">
 
-  <hr style="margin-top: 3rem;"></hr>
+  <hr style="margin-top: 3rem;">
```

Comment:
I know you didn't write this, but while we're focusing on code style do we want to change this over to `class="mt-*"` instead of an inline style?

Reply:
Done in e92f8f0 - I ended up removing the custom spacing entirely.